common/http/HTTPServer.cpp

```diff
@@ -374,6 +374,13 @@ void HTTPResponse::SetNoCache() {
 }
 
 
+/**
+ * @brief Set the appropriate headers so this response is accessible from any origin
+ */
+void HTTPResponse::SetAllowAllOrigin() {
```

Comment:
I'm wondering if we should have a slightly clearer name, perhaps `SetAccessControlAllowAllOrigin` or `SetAccessControlAllowOriginAll`

Reply:
Make sense, will change it to SetAccessControlAllowOriginAll.